src/main/resources/sqlscripts/fn_persistentQ_InsertFirstMultipleCountVal_v1.0.sql

```diff
@@ -0,0 +1,16 @@
+/* CREATE FUNCTION */
+
+CREATE OR REPLACE FUNCTION fn_persistentQ_InsertFirstMultipleCountVal (p_multipleRef varchar(25)) RETURNS void AS $$
+
+BEGIN
+
+LOCK table multipleCounter;
+INSERT into multipleCounter
+SELECT p_multipleRef, 1
+WHERE
+    not exists (select 1 from multipleCounter where multipleref = p_multipleRef);
+
+COMMIT;
```

Comment:
I don't think you need a commit here. We don't have one in any other postgres function. Spring should auto-commit

Reply:
removed